theme/icons/quote.svg

```diff
@@ -1 +1 @@
-<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><g fill="#000" fill-rule="evenodd"><path d="M2.501 9.95L2.5 9.825a6.037 6.037 0 0 1 6.064-6.037L8.557 5.75h-.02A4.076 4.076 0 0 0 4.546 9H8a1 1 0 0 1 1 1v4.5a1 1 0 0 1-1 1H3.5a1 1 0 0 1-1-1V10l.001-.05zM11.001 9.95L11 9.825a6.037 6.037 0 0 1 6.064-6.037l-.007 1.962h-.02A4.076 4.076 0 0 0 13.046 9H16.5a1 1 0 0 1 1 1v4.5a1 1 0 0 1-1 1H12a1 1 0 0 1-1-1V10l.001-.05z"/></g></svg>
\ No newline at end of file
+<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><g fill="#333" fill-rule="evenodd"><path d="M3 10.423a6.5 6.5 0 0 1 6.056-6.408l.038.67C6.448 5.423 5.354 7.663 5.22 10H9c.552 0 .5.432.5.986v4.511c0 .554-.448.503-1 .503h-5c-.552 0-.5-.449-.5-1.003v-4.011-.563zM11 10.423a6.5 6.5 0 0 1 6.056-6.408l.038.67c-2.646.739-3.74 2.979-3.873 5.315H17c.552 0 .5.432.5.986v4.511c0 .554-.448.503-1 .503h-5c-.552 0-.5-.449-.5-1.003v-4.011-.563z"/></g></svg>
```

Comment:
Should it be #333?

Reply:
It should be `#000` but truth to tell, it does not matter because our icons always inherit parent's `color` and we never use them as standalone images.

Reply:
So I'll fix this on master.